hieradata/production.yaml

```diff
@@ -166,6 +166,13 @@ govuk::node::s_backend_lb::backend_servers:
   - 'backend-1.backend'
   - 'backend-2.backend'
   - 'backend-3.backend'
+govuk::node::s_backend_lb::email_alert_api_backend_servers:
+  - 'backend-1.backend'
```

Comment:
Is the idea of this to migrate traffic between both sets of servers for a migration period? Otherwise I don't think the backend servers need to be included.

Reply:
If we just had the new servers in here, would there be an interruption while the apps were deployed, and would anything happen to requests currently in progress? I took this straight from the way publishing-api was moved onto its own machines.

Reply:
Actually, scrap that, since we're removing the apps from the backend machines in `common.yaml`!

Reply:
You're correct in that a smoother migration period may be to run them together at the same time, and then filter down the connections. It's up to you ultimately.

Reply:
Yup, I've set it up to use both the existing backend servers and the new ones, then I'll make up another PR to remove the existing backend servers.

Reply:
👍